encrypter.cpp

```diff
@@ -0,0 +1,168 @@
+#include <iostream>
+#include <fstream>
+#include <regex>
+#include <string>
+#include <stdlib.h>
+#include <unistd.h>
+
+using namespace std;
+
+string read_file(string& address, string error_message, bool clean = false)
+{
+  ifstream f;
+
+  f.open(address);
+  if(!f)
+  {
+    cout << error_message;
+    f.close();
+    exit(1);
+  }
+
+  char c;
+  string text = "";
+
+
+  while (f.get(c))
+    if ((c != ' ') && (c != '\n') && (c != '.') || clean)
+      text += c;
+
+  f.close();
+
+  if (text == "")
+  {
+    cout << "Given file is empty";
+    exit(1);
+  }
+
+  return text;
+}
+
+void record_file(string& address, string& text)
+{
+  ofstream f;
+  f.open(address);
+  f << text;
+  f.close();
+}
+
+string encrypt(string& text, string& key)
+{
+  int shift = rand() % 100;
+  cout << shift << " " << char(shift) << endl;
+
+  for (int i = 0; i < text.length(); i++)
+    text[i] = text[i] ^ key[(i + shift) % key.length()];
+
+  text = char(shift) + text;
+
+  return text;
+}
+
+string decrypt(string& text, string& key)
```

Comment:
But I think it's better to make the key also a binary. You can use a large movie as a key. Why not?